src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml

```diff
@@ -39,9 +39,10 @@
     }
 </div>
 <script at="Foot">
-    $('#@(id)-slider').bootstrapSlider({ tooltip: 'always' }).on('change', function (ev) {
-        $('#@(id)').val(ev.value.newValue);
+    $('#@(id)-slider').bootstrapSlider({ tooltip: 'always' }).on('slide', function (ev) {
```

Comment:
Revert this change, keep what we have

Reply:
Yes, keep 'change' event because 'slide' won't show values while dragging the slider.

Reply:
And I'm guessing that ev.value.newValue is necessary too here.

Reply:
Yes, ev.value is an object that holds both the old and new value in this case.